intf/cnsr_intf.lua

```diff
@@ -201,7 +213,8 @@ function skip(skip_start, skip_end)
 	prev_time = current_time
 	local skip_length = skip_end - skip_start
 	if forward then
-		current_time = math.min(current_time + skip_length + SKIP_SAFETY, vlc.input.item():duration() * MS_IN_SEC) --think if we want to!
+		skip_length=skip_end-math.max(skip_start,current_time)
```

Comment:
?

Reply:
ye, well i thought i fixed it but it does the same as the original idea. so pls dont take those lines